src/components/toast/index.js

```diff
@@ -160,11 +160,17 @@ export default class Toast extends BaseComponent {
   }
 
   getAnimation(shouldShow) {
-    const {position, useNativeDriver} = this.props;
+    const {position, useNativeDriver, animated} = this.props;
     const animationDescriptor = getAnimationDescriptor(position, this.state);
     const {animation, duration, delay} = shouldShow ? animationDescriptor.enter : animationDescriptor.exit;
 
-    return {animation, duration, delay, useNativeDriver, onAnimationEnd: () => this.onAnimationEnd()};
+    return {
+      animation: animated ? animation : null, 
```

Comment:
I would move this condition to the 'animationConfig' state set in the constructor to avoid the call to getAnimation() altogether, as done on line #138.